docs/concepts/database.md

```diff
@@ -121,3 +121,103 @@ The following database versions are required for use with Prefect:
 
 - SQLite 3.24 or newer
 - PostgreSQL 13.0 or newer
+
+
+## Migrations
+
+Prefect uses [Alembic](https://alembic.sqlalchemy.org/en/latest/) to manage database migrations. Alembic is a 
+database migration tool for usage with the SQLAlchemy Database Toolkit for Python. Alembic provides a framework for 
+generating and applying schema changes to a database.
+
+### Running migrations
+To apply migrations to your database you can run the following commands:
+
+To upgrade:
+<div class="terminal">
+```bash
+prefect server database upgrade -y
+```
+</div>
+To downgrade:
+<div class="terminal">
+```bash
+prefect server database downgrade -y
+```
+</div>
+
+You can use the `-r` flag to specify a specific migration version to upgrade or downgrade to. 
+For example, to downgrade to the previous migration version you can run:
+<div class="terminal">
+```bash
+prefect server database downgrade -y -r -1
+```
+</div>
+or to downgrade to a specific revision:
+<div class="terminal">
+```bash
+prefect server database downgrade -y -r d20618ce678e
+```
+</div>
+
+### Adding a new migration
+To make changes to a table, first update the SQLAlchemy model in `src/prefect/server/database/orm_models.py`. For example
+if you wanted to add a new column to the `flow_run` table, you would add a new column to the `FlowRun` model:
```

Comment:
I think this section should probably be in the contributing docs

Reply:
can definitely shift this over there. do you think it makes sense to link to that from this page at all?

Reply:
Can't hurt to link :)